js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js

```diff
@@ -340,13 +313,14 @@
         });
 
         arangoHelper.initSigma();
-      }.bind(this);
+      }).bind(this);
 
-      $(window).resize(function () {
+      // TODO: Should a UI function be part of the router?
```

Comment:
No it shouldn't :-) 
Let's get rid of those problems as soon as we can get rid of the Backbone router.
But I bet this problem will be solved anyway, as soon as we can deprecate Backbone views - at least the way how they are currently implemented.

Reply:
Agreed. This issue will go away when the Backbone router is deprecated. No action needed for now 👍